cms/djangoapps/contentstore/views/preview.py

```diff
@@ -247,30 +247,30 @@ def get_user_group_id_for_partition(self, user, user_partition_id):
         return None
 
 
-def _load_preview_block(request, descriptor):
+def _load_preview_block(request, block):
     """
-    Return a preview XBlock instantiated from the supplied descriptor. Will use mutable fields
+    Return a preview XBlock instantiated from the supplied block. Will use mutable fields
     if XBlock supports an author_view. Otherwise, will use immutable fields and student_view.
 
     request: The active django request
-    descriptor: An XModuleDescriptor
+    block: An XModule Block
```

Comment:
```suggestion
    block: An XModuleMixin
```

A quick explanation of why we don't want to use `XBlock` - this method uses `block.bind_for_student`, which is specific to the `XModuleMixin`.

Reply:
Updated.